docs/manuals/source/Configuration/Director.rst

```diff
@@ -1151,21 +1151,38 @@ The directives within an Options resource may be one of the following:
 .. config:option:: dir/fileset/include/options/HardLinks
 
    :type: yes|no
-   :default: yes
+   :default: no 
+
+   New default since :sinceVersion:`23.0.0: Bareos > 23.0.0`
+
+   When disabled Bareos will most likely backup more data and the system will 
+   not be restored identically to the original. The fact that file is a hardling
+   will be missing.
+
+   When enabled, this directive will cause hard links to be backed up as hard links.
+   For each set of hardlinks, the file daemon will only backup the file contents once --
+   when it encounters the first file of that set -- and only backup meta data and a
+   reference to that first file for each subsequent file in that set. 
+
+   Be aware that the process of keeping track of the hard links can be quite 
+   expensive if you have lots of them (tens of thousands or more). Backups become 
+   very long and the File daemon will consume a lot of CPU power checking hard links.
+   In such a case, set :config:option:`dir/fileset/include/options/HardLinks = no`
+   and hard links will not be backed up.
```

Comment:
Do you have any expierences with this ?  I tried creating 600k sets of 3 hardlinked files each and the version that with `HardLinks=yes` was almost twice as fast as the version with `Hardlinks=no`.  Im not sure if this is still true (if it ever was!)